packages/dx-react-scheduler/src/plugins/day-view.tsx

```diff
@@ -48,13 +45,43 @@ const DayScaleEmptyCellPlaceholder = () => <TemplatePlaceholder name="dayScaleEm
 const DayScalePlaceholder = () => <TemplatePlaceholder name="dayScale" />;
 const TimeScalePlaceholder = () => <TemplatePlaceholder name="timeScale" />;
 
-class DayViewBase extends React.PureComponent<VerticalViewProps, ViewState> {
-  timeTable = React.createRef<HTMLElement>();
-  layout = React.createRef<HTMLElement>();
-  layoutHeader = React.createRef<HTMLElement>();
+const scrollingStrategyComputed = memoize((viewName, scrollingStrategy) => getters =>
```

Comment:
Computed function memoization should be done in context of Plugin instance (not in global context).

Because we want a particular Plugin instance to use same function instance as long as possible.

This code says that we want **ALL** Plugin instances (in **ALL** Schedulers on a page) to use same function instance as long as possible.
First, it is not what we want.
Second, most likely memoization cache will be discarded on each function call.

----
`memoize` keeps only last result and that result belongs (should actually belong) to a particular Plugin instance.

Reply:
@DmitryBogomolov great note! Thank you very much!